# sarscov2-analysis-resources

This is a list of links to websites and software that can be used for SARS-CoV-2 sequence analysis and epidemiology of strains / mutations.  
Not included are dashboards that show general SARS-CoV-2 statistics, e.g., the number of cases per region, etc.

## Sequence analysis
* [About the PANGO nomenclature ](https://www.pango.network/)
  * [List of PANGO lineages](https://cov-lineages.org/lineage_list.html)
* [Pangolin web server](https://pangolin.cog-uk.io/) for assigning PANGO lineages to assembled genomes
* [Nextclade](https://clades.nextstrain.org/), clade assignment, mutation calling, and sequence quality checks
* [Nextstrain](https://nextstrain.org/)
* [UShER](https://genome.ucsc.edu/cgi-bin/hgPhyloPlace), place sequences in a global phylogenetic tree
* [Gensplore MN908947](https://gensplore.theo.io/?gb=%2Fsequence.gb), browse the reference genome
* [Cov2Tree](https://cov2tree.org/)

## Geographic distribution of mutations and lineages
* [Outbreak.info](https://outbreak.info/)
  * [List of VOCs](https://outbreak.info/situation-reports)
  * [Lineage comparison by mutations](https://outbreak.info/compare-lineages)
* [CoVizu](https://filogeneti.ca/CoVizu/)
* [CoV-Spectrum](https://cov-spectrum.org/explore/World/AllSamples/Past6M), easy queries for filtering by strains and/or mutations
  * [Variant query example](https://cov-spectrum.org/explore/Europe/AllSamples/Past3M/variants?variantQuery=%21B.1.1.529+%26+S%3A69-)
* [CoVariants](https://covariants.org/)
* [COVID-19 virus mutation tracker](https://www.cbrc.kaust.edu.sa/covmt/)
* [COG-UK Mutation Explorer](https://sars2.cvr.gla.ac.uk/cog-uk/)

## Software for download
* [Pangolin](https://github.com/cov-lineages/pangolin), also via bioconda
* [Nextclade CLI](https://docs.nextstrain.org/projects/nextclade/en/stable/user/nextclade-cli.html), also via bioconda
* [connor-lab/ncov2019-artic-nf](https://github.com/connor-lab/ncov2019-artic-nf), nextflow-based pipeline for generating consensus sequences from amplicon-sequencing

## Databases
* [NCBI SARS-CoV-2 Resources](https://www.ncbi.nlm.nih.gov/sars-cov-2/)
  * [SARS-CoV-2 Data Hub](https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/virus?SeqType_s=Nucleotide&VirusLineage_ss=SARS-CoV-2,%20taxid:2697049)
  * [SARS-CoV-2 Variants Overview](https://www.ncbi.nlm.nih.gov/activ)
* [GISAID](https://gisaid.org/)

## Literature
* [CoronaCentral.ai](https://coronacentral.ai/), literature search for SARS-CoV-2 and other viruses

## Other link collections
* [CDC SARS-CoV-2 Sequencing Resources](https://github.com/CDCgov/SARS-CoV-2_Sequencing)
* [soroushchehresa/awesome-coronavirus](https://github.com/soroushchehresa/awesome-coronavirus)
